Source/Core/VideoBackends/D3D12/Render.cpp

```diff
@@ -1244,6 +1242,8 @@ void Renderer::SetSamplerState(int stage, int tex_index, bool custom_tex)
 	if (tex_index)
 		stage += 4;
 
+	const auto original_hex = gx_state.sampler[stage].hex;
```

Comment:
`hex` likely won't be changed, since it maps a hardware construct (IMO, we really shouldn't be using a union and struct bitfield combo like that anyway, since it makes structs non-trivial, making memcpy and memmove unsafe on them, for essentially zero gain over providing proper accessors)